src/app/home/home.css

```diff
@@ -0,0 +1,131 @@
+body {
+    margin: 0;
+    padding: 0;
+}
+
+#landingPage {
+    height: 85%;
```

Comment:
why do you have both css and scss file?

Reply:
Where exactly? I only added home.css to style the homepage component